examples/dapperish/dapper.go

```diff
@@ -1,4 +1,4 @@
-package main
+package dapperish
```

Comment:
I had to rename the package to be able to import it into tests, since we don't have any other working implementation of the trace context source.